stories/__tests__/SQFormIconButton.stories.test.js

```diff
@@ -12,12 +12,23 @@ const {
 
 describe('SQFormIconButton Tests', () => {
   describe('Button Only', () => {
+    it('should render a button with an icon', () => {
+      render(<SQFormIconButton exampleIcons={CheckCircle} />);
+
+      const iconButton = screen.getByRole('button', /form submission/i);
+      const svg = document.getElementsByTagName('svg');
+
+      expect(iconButton).toBeInTheDocument();
+      expect(svg.length).toEqual(1);
```

Comment:
@codyarose This is the closest I could get it to test for the icon. The <svg> is `area-hidden`, doesn't have a role, and doesn't have any text.

Reply:
@laurelbean @codyarose I'm going to merge this PR, then i'll add a comment to #228 that we should fix this test as part of the PR that resolves issue #228